src/app.js

```diff
@@ -17,22 +22,30 @@ const inputBuilder = {
     return item;
   },
 
-  run: function(){
+  prepare: function(){
     const outerContainer = this.buildElement('div', { id: 'items-outer-container'});
     const container = this.buildElement('div', { id: 'items-container'});
     const itemInput = this.buildElement('div', { id: 'item-input'});
-    const textInput = this.buildElement('input', { type: 'text', placeholder: 'Add a new task'});
+    const todo = document.getElementById('todo-list');
 
     outerContainer.append(container);
     container.append(itemInput);
-    itemInput.append(textInput)
+    todo.append(outerContainer);
 
-    const todo = document.getElementById('todo-list');
+    return itemInput;
+  },
 
-    todo.append(outerContainer);
+  createInput: function(attributes){
+    const itemInput = this.prepare();
+    const newInput = new InputText(attributes);
```

Comment:
Adicionando os métodos na herança do objeto principal, o `new` será instanciado do lado de fora, e você poderá usar o valor do `input`, através do `this`.